changelogs/unreleased/6604-izturn-minor.md

```diff
@@ -0,0 +1,4 @@
+
+## Customize the certificate's lifetime
+
+customize the number of days for which certificates will be valid. defaults to 365.
```

Comment:
It could be mentioned that this concerns deployments that use Gateway Provisioner and that the expiry is customized by `ContourDeployment.spec.certLifetime`.